tests/integration_tests/functional/test_drives.py

```diff
@@ -92,9 +92,11 @@ def test_rescan_dev(test_microvm_with_ssh, network_config):
         size=512
     )
 
-    losetup = ['losetup', '--find', '--show', fs2.path]
-    losetup_result = run(losetup, stdout=PIPE, check=True)
-    loopback_device = losetup_result.stdout.decode('utf-8').rstrip()
+    # "losetup --find" may slow down tests due to
+    # https://github.com/firecracker-microvm/firecracker-containerd/issues/141
+    loopback_device = f'/dev/loop{os.getpid()}'
```

Comment:
Should we assume that it is safe to do not care about the scenario when `Device is busy`, which can happen when '/dev/loop{os.getpid()}' already exists? IMO it is a very rare case, but just wanted to make a note about it, if anybody knows a way to avoid it, otherwise, I think it does not worth any effort to change it.

Reply:
One way we can avoid the scenario is using `losetup --find` again. firecracker-containerd is using the command a lot and it caused the performance regression, but Firecracker is using the command only once. I think we can prioritize correctness over performance concerns.